src/typings/createTransform.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module 'redux-persist/createTransform' {
+  type TransformConfig = {
+    whitelist?: string[],
+    blacklist?: string[],
+  }
+
+  type Transformer = {
```

Comment:
Id rather define this as generic `Transform<State, Raw>` the same way as it was in the v4. At least we won't break types for depending libraries such as `redux-persist-transform-filter` etc.

Reply:
Hi @LKay, I have created a pull request today which has the entire definition for `redux-persist` though is not split into multiple files.

In this pull request, the transformer is a generic definition, do check it here: #624

Reply:
@LKay 
```ts
   export function persistCombinedReducers<State>(config: PersistConfig, reducers: ReducersMapObject): Reducer<State>;
```

should include the state coming from redux_persist itself, so it should be

```ts
   export function persistCombinedReducers<State>(config: PersistConfig, reducers: ReducersMapObject): Reducer<State & PersistedState>;
```